pg_search/src/gucs.rs

```diff
@@ -161,14 +164,25 @@ pub fn init() {
     GucRegistry::define_float_guc(
         c"paradedb.limit_fetch_multiplier",
         c"Multiplier for the limit in a TopN query",
-        c"Multiplier for the limit in a TopN query",
+        c"The limit is multiplied by this factor to determine the chunk size, and subsequent retries will multiply the chunk size by this factor. A higher value reduces the probability of a re-query for a TopN query but increases query times.",
```

Comment:
This is finicky. Don't need to play with it too much now. If it ends up being something that more users need to tune, it'll deserve a docs section probably.

```suggestion
        c"The limit is multiplied by this factor to determine the chunk size, and subsequent retries will multiply the chunk size by this factor. A higher value reduces the probability of a re-query for a TopN query but increases per-query times.",
```